seasons.py

```diff
@@ -1,21 +1,17 @@
-def season (month):  # remove whitespace `def season(month):`
+def season(month):  # remove whitespace `def season(month):`
 
-    if 0<month<2 or month==12:  # add whitespaces `if 0 < month < 2 or month == 12:`
-        print('Now is winter')  # use return instead print
-    elif 3<=month<6:
-        print('Now is spring')
-    elif 6<=month<9:
-        print("Now is autumn")
-    elif 9<=month<12:
-        print("Now is winter")
+    if 0 < month < 2 or month == 12:  # add whitespaces `if 0 < month < 2 or month == 12:`
+        return ('Now is winter')  # use return instead print
```

Comment:
Return just a season name from the function and format it to make a sentence somewhere else